src/Cli/dotnet/commands/dotnet-workload/ToolCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
Did you mean to rename this file? WorkloadCommand.cs instead of ToolCommand.cs? Same for most other command files, they still have tool in the file names.

Reply:
renamed